x/proof/keeper/msg_server_create_claim_test.go

```diff
@@ -284,10 +285,10 @@ func newTestClaimMsg(
 		supplierAddr,
 		&sessiontypes.SessionHeader{
 			ApplicationAddress:      appAddr,
-			SessionId:               sessionId,
 			Service:                 service,
+			SessionId:               sessionId,
 			SessionStartBlockHeight: 1,
-			SessionEndBlockHeight:   4,
+			SessionEndBlockHeight:   sessionkeeper.GetSessionEndBlockHeight(1),
```

Comment:
... like here (i.e. not calculating session end height).

Reply:
Why do you not want to calculat session end height?

Reply:
Resolving but reopen if it's still a concern.